### PR TITLE
[SIG-4482]_styling_checkbox_and_checkboxlabel_changed_in_StatusForm

### PR DESCRIPTION
**Pull request by SireeKoolenWijkstra**
Styling needed to change of the checkbox and the label surrounding the checkbox. The checkbox needed to be opaque and the label needed to be color #767676. I

n StatusForm I changed a label to newly created StyledCheckboxLabel and changed Checkbox to newly created StyledCheckbox. In styled.tsx opacity: 0.3 added to styling of StyledCheckbox, and color: #767676 to StyledCheckboxLabel.

### CODE REVIEW

File: src/signals/incident-management/containers/IncidentDetail/components/StatusForm/StatusForm.tsx [hunk 1]
```diff
@@ -339,20 +340,20 @@ const StatusForm: FunctionComponent<StatusFormProps> = ({
...
+              <StyledCheckboxLabel
                 disabled={state.check.disabled}
                 htmlFor="send_email"
                 label={constants.MELDING_CHECKBOX_DESCRIPTION}
                 noActiveState
               >
-                <Checkbox
+                <StyledCheckbox
                   checked={state.check.checked}
                   data-testid="sendEmailCheckbox"
```
Comment (OscarBakker) [inline src/signals/incident-management/containers/IncidentDetail/components/StatusForm/StatusForm.tsx:351]: - [ ] doen

**Comment by kiruys** [comment]
Would you mind to clean up the title of the PR by removing the underscores? 🤓

**Review by kiruys — APPROVED** [review]
The code looks good, Siree!

**Comment by SireeKoolenWijkstra** [comment]
Hi @kiruys,

The same title of the PR but now without underscores. Life is good again.

**Pull request merged**